cclib/parser/psi4parser.py

```diff
@@ -75,7 +75,7 @@ def extract(self, inputfile, line):
             # Keep track of early versions of Psi4.
             if "beta" in line:
                 self.version_4_beta = True
-        self.metadata["package_version"] = 4
+        self.metadata["package_version"] = "4"

```

Comment:
If there is type enforcement/conversion, it should be done in a similar way to the regular attributes.

Reply:
In fact maybe `metadata` should be a class like `ccData`.

Reply:
I dunno. I think it's sufficient that it's just another attribute. When we expand the scope of attribute to be custom object (for 2.x) this will make it easier to do arbitrary stuff for metadata. WDYT?

Reply:
Good point, because a piece of metadata should always be associated with an output, and the flexible structure would give the same result anyway.